qualtran/bloqs/rotations/quantum_variable_rotation.ipynb

```diff
@@ -142,7 +154,7 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "32069e11",
+   "id": "c141bb25",
```

Comment:
do you know how all these ids got changed? would it be possible to revert these?

Reply:
I regenerated the notebook since the docstrings are updated, maybe that triggered the id change? I'm not sure. Is it worth manually reverting it?